client/app.jsx

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
-import Home from './pages/home';
+// import Home from './pages/home';
+import WorkoutSession from './pages/workout-session';
```

Comment:
Be sure to always remove all commented out code before submitting a pull request.